eva/catalog/models/df_column.py

```diff
@@ -81,7 +81,22 @@ def array_dimensions(self):
 
     @array_dimensions.setter
     def array_dimensions(self, value: List[int]):
-        self._array_dimensions = str(value)
+        # backward compatibility
+        if not isinstance(value, list):
+            self._array_dimensions = str(value)
```

Comment:
Could you remind me what is the backward compatibility for?

Reply:
Fixed